src/Microsoft.DotNet.Darc/Darc/Operations/GatherDropOperation.cs

```diff
@@ -1434,6 +1446,7 @@ private static string GetBlobBaseUri(string blobUri)
                     sourceUri,
                     Logger,
                     authHeader: authHeader,
+                    adjustRequestMessage: AdjustRequestMessage,
```

Comment:
I am thinking whether this should not go in the `ExecuteAsync` instead? I guess it doesn't matter all that much

Reply:
Could be for some complex scenario. Not sure if the token assignment is the one.